media-libs/fcft/fcft-2.5.1.ebuild

```diff
@@ -0,0 +1,85 @@
+# Copyright 2020-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit meson
+
+if [[ ${PV} != *9999* ]]; then
+	SRC_URI="https://codeberg.org/dnkl/fcft/archive/${PV}.tar.gz  -> ${P}.tar.gz"
+	KEYWORDS="~amd64"
+	S="${WORKDIR}/${PN}"
+else
+	inherit git-r3
+	EGIT_REPO_URI="https://codeberg.org/dnkl/fcft.git"
+fi
+
+DESCRIPTION="A simple library for font loading and glyph rasterization"
+HOMEPAGE="https://codeberg.org/dnkl/fcft"
+LICENSE="MIT"
+SLOT="0/3"
+IUSE="examples +text-shaping +man test"
+RESTRICT="!test? ( test )"
+
+RDEPEND="
+	dev-libs/tllist
+	media-libs/fontconfig
+	media-libs/freetype
+	x11-libs/pixman
+	text-shaping? (
+		dev-libs/libutf8proc:=
+		media-libs/harfbuzz:=
+	)
+	examples? (
+		dev-libs/libutf8proc:=
+		dev-libs/wayland
+	)
+"
+DEPEND="
+	${RDEPEND}
+	test? (
+		dev-libs/check
+		text-shaping? ( media-fonts/noto-emoji )
+	)
+"
+BDEPEND="
+	app-i18n/unicode-data
+	man? ( app-text/scdoc )
```

Comment:
I shouldn't have a IUSE=man over scdoc personally, plus foot already pulls it unconditonally.